umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -37,12 +68,7 @@
    (if (and (nil? description-index)(nil? url-content-type-index)
             (nil? type-index) (nil? subtype-index))
     {:Description description} ; Description not formatted like above, so just description
-    {:Description (when description-index
-                   (let [desc (subs description
-                               description-index
-                               (or url-content-type-index type-index subtype-index
-                                   checksum-index (count description)))]
-                    (str/trim (subs desc (inc (.indexOf desc ":"))))))
+    {:Description (convert-description-string-to-map description description-index)
      :URLContentType (when url-content-type-index
```

Comment:
You will need to add your code also to :URLContentType, :Type, :Subtype, and :Checksum below. They have the same problem.